CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Unreleased
 
+## 0.8.2 (May 3, 2018)
```

Comment:
Sorry nitpick, can you remove this heading and just leave it under `unreleased`. 

We'll add the version header in a version bump commit like: https://github.com/procore/ruby-sdk/commit/58582e5bf57cc2b5fc4d684f2c74e92797c3aa19

Reply:
sounds good 👍